packages/client/src/index.ts

```diff
@@ -58,6 +62,7 @@ export class Client {
       dimensions: (_injector: Injector) => {
         throw new Error('Dimension service does not exist yet');
       },
+      logger: (_injector: Injector) => debug('yavin:client'),
```

Comment:
Can we consolidate this string with the one in packages/data/addon/services/yavin-client.ts